packages/neuron-wallet/src/main.ts

```diff
@@ -20,6 +20,8 @@ const initUILayer = (win: BrowserWindow) => {
   })
   // switch language according to locale
   setLanguage(win, app.getLocale())
+  // send transaction history on window load
```

Comment:
Don't feel the comment is necessary. It just duplicates the function call - the function name already describes what it does well.

So is the switch language comment.

Reply:
comments removed